tests/drivers/can/api/src/classic.c

```diff
@@ -486,6 +486,24 @@ ZTEST_USER(can_classic, test_bitrate_limits)
 	zassert_true(min <= max, "min bitrate must be lower or equal to max bitrate");
 }
 
+/**
+ * @brief Test setting a too low bitrate.
+ */
+ZTEST_USER(can_classic, test_set_bitrate_too_low)
+{
+	uint32_t min = can_get_bitrate_min(can_dev);
+	int err;
+
+	err = can_stop(can_dev);
+	zassert_equal(err, 0, "failed to stop CAN controller (err %d)", err);
+
+	err = can_set_bitrate(can_dev, min - 1);
```

Comment:
Same as above.

Reply:
I added ztest_test_skip() when minimum bitrate is 0.